net/https-dns-proxy/Makefile

```diff
@@ -1,17 +1,16 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=https_dns_proxy
-PKG_VERSION:=2018-04-23
-PKG_RELEASE=4
+PKG_RELEASE=1
 
-PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
-PKG_MIRROR_HASH:=24b7e4238c37e646f33eee3a374f6b7beb5c167b9c5008cc13b51e5f1f3a44ea
-PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
-PKG_SOURCE_URL:=https://github.com/aarond10/https_dns_proxy/
 PKG_SOURCE_PROTO:=git
-PKG_SOURCE_VERSION:=bea68401330e611f6e9b75cec84e2dc4e81e52de
+PKG_SOURCE_URL:=https://github.com/aarond10/https_dns_proxy
+PKG_SOURCE_DATE:=2019-10-09
+PKG_SOURCE_VERSION:=3d20c75c4a17aa569748e63ac169b9274178d573
+PKG_MIRROR_HASH:=149839f1db4ab9691db3955de526080a5b731ff8484b06f164497f4e244140da
+
 PKG_MAINTAINER:=Aaron Drew <aarond10@gmail.com>
-PKG_LICENSE:=MIT
```

Comment:
Why did you removed the PKG_LICENSE?

Reply:
A misunderstanding that PKG_LICENSE_FILES replaced PKG_LICENSE.  Now fixed.